test/externalTests.sh

```diff
@@ -91,10 +91,15 @@ function test_truffle
 module.exports['compilers'] = {solc: {version: "$DIR/solc"} };
 EOF
 
-      npx truffle compile
-      echo "Verify that the correct version ($SOLCVERSION) of the compiler was used to compile the contracts..."
-      grep -e "$SOLCVERSION" -r build/contracts > /dev/null
-      npm run test
+      for optimize in "{enabled: false }" "{enabled: true }" "{enabled: true, details: { yul: true } }"
+      do
+        rm -rf build || true
+        echo "module.exports['compilers']['solc']['settings'] = {optimizer: $optimize };" >> truffle*.js
```

Comment:
Did you verify this passes through to jsonio?

Reply:
I hoped there would be some gas values, but I'll add a check to re-discover this in the generated metedata like we do with the compiler version above.

Reply:
Hm, it looks like this is not possible. I'll check manually on circle.